apps/src/codebridge/FileBrowser/dragAndDropUtils.ts

```diff
@@ -64,11 +69,12 @@ export const fileBrowserKeyboardCoordinateGetter: (
   });
   const currentIndex = orderedRects.indexOf(over?.id as string);
   let nextIndex = currentIndex;
-  // Find the next index based on the key pressed.
-  if (event.code === KeyboardCode.Down) {
-    nextIndex = Math.min(currentIndex + 1, orderedRects.length - 1);
+  // Find the next index based on the key pressed. We cycle around if they reach
+  // the beginning or end of the list.
+  if (event.code === KeyboardCode.Down || event.code === KeyboardCode.Tab) {
+    nextIndex = (currentIndex + 1) % orderedRects.length;
   } else if (event.code === KeyboardCode.Up) {
-    nextIndex = Math.max(currentIndex - 1, 0);
+    nextIndex = (currentIndex - 1) % orderedRects.length;
```

Comment:
I think `nextIndex` is set to `-1` when `currentIndex` is `0`. Do we want `nextIndex = (currentIndex - 1 + orderedRects.length) % orderedRects.length`?

Reply:
The `%` operator does this for us, it's the remainder of the division. If we have 4 rects, the possible options in the operations we do here and in the other case are:
-  `-1 % 4 = 3`
- `0 % 4 = 0`
-  `1 % 4 = 1`
-  `2 % 4 = 2`
-  `3 % 4 = 3`
-  `4 % 4 = 0`

So we will always end up with a `nextIndex` between 0 and 3, which is what we want.

Reply:
thanks for catching this!

Reply:
Yeah, JS handles modulo differently than taught in math class which is annoying.